RELEASE-NOTES.md

```diff
@@ -183,7 +187,6 @@ Changes in the `Entity` hierarchy:
 * Changed the constructor signature of `Property`
 * Removed `Entity::setClaims` (`Item::setClaims` has been retained)
 * Removed `Entity::stub`
-* Removed `Property::newEmpty`
```

Comment:
The relevant release here is 3.0, not 1.0. It was removed in 3.0 as far as I can tell: https://github.com/wmde/WikibaseDataModel/blob/3.0.0/src/Entity/Property.php